pkg/sbom/cyclonedx/marshal.go

```diff
@@ -59,23 +60,27 @@ func NewMarshaler(version string, opts ...core.Option) *Marshaler {
 // Marshal converts the Trivy report to the CycloneDX format
 func (e *Marshaler) Marshal(report types.Report) (*cdx.BOM, error) {
 	// Convert
-	root, err := e.MarshalReport(report)
+	root, err := MarshalReport(report)
 	if err != nil {
 		return nil, xerrors.Errorf("failed to marshal report: %w", err)
 	}
 
 	return e.core.Marshal(root), nil
 }
 
-func (e *Marshaler) MarshalReport(r types.Report) (*core.Component, error) {
+func (e *Marshaler) MarshalKbom(report k8s.Report) (*cdx.BOM, error) {
```

Comment:
I don't want to add MarshalKbom here. Kubernetes should directly call core.Marshal.

Reply:
we might need to add this function :
```
func (e *Marshaler) MarshalComponent(component *core.Component) (*cdx.BOM, error) {
	return e.core.Marshal(component), nil
}
```
the other option is to expose  `Marshaler.core`

what do you prefer?